### PR TITLE
Feature/generate changelog readme

### LINKED ISSUE #2: ⚠️ Greenkeeper is no longer available for new installations
Hello!

**Greenkeeper is no longer available for new installations.**

The service will be saying goodbye 👋 and passing the torch to Snyk on June 3rd, 2020. [Find out more at greenkeeper.io](https://greenkeeper.io).

If this is your only Greenkeeper installation, you can just [sign up for Snyk directly](https://snyk.io), it’s free for Open Source even has free features for private projects as well.

Nevertheless, thanks for your interest in Greenkeeper! We’re sure your repositories will find a good home at Snyk ☀️🏡💜

All the best from everyone at Greenkeeper! 👋🤖🌴

### LINKED ISSUE #3: ⚠️ Greenkeeper is no longer available for new installations
Hello!

**Greenkeeper is no longer available for new installations.**

The service will be saying goodbye 👋 and passing the torch to Snyk on June 3rd, 2020. [Find out more at greenkeeper.io](https://greenkeeper.io).

If this is your only Greenkeeper installation, you can just [sign up for Snyk directly](https://snyk.io), it’s free for Open Source even has free features for private projects as well.

Nevertheless, thanks for your interest in Greenkeeper! We’re sure your repositories will find a good home at Snyk ☀️🏡💜

All the best from everyone at Greenkeeper! 👋🤖🌴

### PR DESCRIPTION
**Pull request by NathanDowner**
**Summary of Changes**

- created function to generate a CHANGELOG.md file based on the change log data in the changelog.ts file in the modules folder.

**Related Issues**

**Additional Context**

**Screenshots/Videos**
image

**Testing Steps**

---

**Checklist**

-   [ ] Sufficient unit tests added
-   [x] Sufficient logging added
-   [x] Sufficient in-line code documentation

### CODE REVIEW

File: src/lib/utilities.ts [hunk 1]
```diff
@@ -366,6 +366,53 @@ function checkProjectValidity(): { isValid: boolean, projectRoot: string } {
...
+      return `${resource.destPath}/${resource.name}`;
+    }
+
+    return `${resource.name}`;
+  });
+
+  const updatedFiles: string[] = updatedChangeLogResources.map(resource => (resource.destPath));
+
+  const readmeContent = `# Changelog - ${versionName}
+The ${'`upgrade`'} command is used to upgrarde a project to the latest version of the template, or to a speciied version.
```
Comment (Saspect-IO) [inline src/lib/utilities.ts:402]: not sure whats happening here `${'`upgrade`'} ` seems like its just going to be the string `upgrade`

**Comment by Saspect-IO** [comment]
Thanks @NathanDowner, great job on the pr. 
Please edit and add description to pr above with a screenshot of generated result. 
I have added an outline for guidance.

File: src/lib/utilities.ts (hunk 1 above)
Comment (NathanDowner) [reply src/lib/utilities.ts:402]: Yea that's correct. it was just a way of escaping the backticks. At the time I forgot about using a backslash. I'll update it

File: src/lib/utilities.ts [hunk 2]
```diff
@@ -365,6 +365,62 @@ function checkProjectValidity(): { isValid: boolean, projectRoot: string } {
...
+      return `${resource.destPath}/${resource.name}`;
+    }
+
+    return `${resource.name}`;
+  });
+
+  const updatedFiles: string[] = updatedChangeLogResources.map(resource => (resource.destPath));
+
+  const readmeContent = `# Changelog - ${versionName}
+The \`upgrade\` command is used to upgrarde a project to the latest version of the template, or to a speciied version.
```
Comment (natvrey) [inline src/lib/utilities.ts:409]: Line 409 the 2nd 'upgrade' has a typo.

Comment (NathanDowner) [reply src/lib/utilities.ts:409]: Great catch, thanks Nat!

**Pull request merged**